azurerm/validators.go

```diff
@@ -246,3 +246,26 @@ func validateFilePath() schema.SchemaValidateFunc {
 		return
 	}
 }
+
+//Following the naming convention as laid out in the docs https://msdn.microsoft.com/library/azure/dn167011.aspx
+func validateArmStorageShareName(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
since this is the only place it's used, can we move this back into the share resource?

Reply:
Done